hack/ollama_metrics/ollama_gpu.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ollama
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ollama
+  namespace: ollama
+spec:
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      name: ollama
+  template:
+    metadata:
+      labels:
+        name: ollama
+      annotations:
+        prometheus.io/scrape: true
+        prometheus.io/path: "/metrics"
+        prometheus.io/port: 8445
+    spec:
+      containers:
+      - name: ollama
+        image: ollama-metrics:latest
+        imagePullPolicy: Never
+        env:
+        - name: PATH
```

Comment:
Question. Is this required? I thought nvidia container runtime does all necessary setup.

Reply:
Not sure, probably not needed. The yaml file is based on https://github.com/ollama/ollama/blob/main/examples/kubernetes/README.md, which requires Nvidia [k8s-device-plugin](https://github.com/NVIDIA/k8s-device-plugin) only.

Reply:
I see. Thanks! Let me try.